src/controllers/module.controller.js

```diff
@@ -92,20 +92,22 @@ const updateModuleById = async (req, res, next) => {
         }
       )
 
-      updatedModule.addActions(action_id)
       if (updatedModule.length) {
         return res.json({ message: 'Module updated successfully' })
       }
     }else{
-      const updated = await Module.update(
-        { name, url, icon, module_id },
+      let [count, updateModule] = await Module.update(
+        { name, url, icon, module_id, action_id },
         {
           where: {
             id,
           },
+          returning :true
         }
       )
-      if (updated.length) {
+
+      updateModule[0].setActions(action_id)
+      if (updateModule.length) {
```

Comment:
Aqui tambien el count.
```suggestion
      if (count {
```

Reply:
Listo lo voy a arreglar